HAWK-11: add readme (#15)
codex-team/hawk.android: README.md

> # HAWK

Reviewer: ```suggestion
# Hawk Android
```

> It is mobile adaptation of web site for [hawk.so](https://hawk.so)

Reviewer: ```suggestion
This is a mobile client for the [Hawk](https://hawk.so) errors monitoring system
```

The app is written in Kotlin

## Dependencies
We used next dependencies in application:
* RxJava 3.0
* Apollo GraphQL

## Development Environment
Download and install Android Studio or Intellij IDEA

## How to run
If you have a phone with the android OS:
  * Download the application from releases on github or telegram channel.
  * Allow the app to be installed from an unknown source in settings. ([link](https://www.androidinfotech.com/unknown-sources-app-installation-android/))
  * Disable system protector in google play, if it need. ([link](https://support.mobile-tracker-free.com/hc/en-us/articles/360005346953-How-to-disable-Google-Play-Protect-))

>   * Download apk from releases in github or telegram channel

Reviewer: этот пункт уже был

  * Install and launch the application

If you don't have any devices with android OS:

>   * Download and install [BlueStack](https://www.bluestacks.com/ru/index.html) or you can try to install "Command line tools" from Android studio with out development enviroment and use sdk manager for install android virtual devices and use them for testing the application.

Reviewer: >  with out development enviroment

что это значит?

>   * Download and install [BlueStack](https://www.bluestacks.com/ru/index.html) or you can try to install "Command line tools" from Android studio with out development enviroment and use sdk manager for install android virtual devices and use them for testing the application.

Reviewer: сложно как-то. Если тут несколько пунктов, разбей их на подпункты

>   * Download apk from releases in github or telegram channel

Reviewer: ```suggestion
  * Download the APK from GitHub releases or telegram channel
```

>   * Drag apk on emulator, waiting after install

Reviewer: ```suggestion
  * Drag the APK to the emulator, wait for the installation
```
